From cbc75dc98d7d57ce6c4e137146142707ecbcf754 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Dylan=20A=C3=AFssi?= <dylan.aissi@collabora.com>
Date: Wed, 14 Dec 2022 11:07:42 +0100
Subject: [PATCH] pw-mon: recognize -N and -C as valid options

---
 src/tools/pw-mon.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/tools/pw-mon.c b/src/tools/pw-mon.c
index f7e088c65..2eeb4dd0e 100644
--- a/src/tools/pw-mon.c
+++ b/src/tools/pw-mon.c
@@ -780,7 +780,7 @@ int main(int argc, char *argv[])
 	if (isatty(STDERR_FILENO) && getenv("NO_COLOR") == NULL)
 		colors = true;
 
-	while ((c = getopt_long(argc, argv, "hVr:", long_options, NULL)) != -1) {
+	while ((c = getopt_long(argc, argv, "hVr:NC", long_options, NULL)) != -1) {
 		switch (c) {
 		case 'h':
 			show_help(argv[0], false);
-- 
GitLab

